javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,31 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (!studentName) {
+    alert('"please enter a Name"');
```

Comment:
Is there a reason you are using both single and double quotes here ? There is no need to quote your string inside of quotes, unless you really want those "" to show🏋️

Reply:
yes, i wanted to show text with double quotes when alert show up in browser. looks a bit fancy i think :)